src/System/Random/Internal.hs

```diff
@@ -956,7 +956,7 @@ class Uniform a where
   --
   -- There is a default implementation via 'Generic':
   --
-  -- >>> :set -XDeriveGeneric -XDeriveAnyClass
+  -- >>> :seti -XDeriveGeneric -XDeriveAnyClass
```

Comment:
@Bodigrim `:set` sets options for source files and `ghci` expressions, while `:seti` only affects `ghci` expressions.

1. With `:set` those three tests were failing, I think due to a reincarnation of https://gitlab.haskell.org/ghc/ghc/-/issues/20670.
1. For doctests you usually want `:seti` anyway.